### PR TITLE
Changes in side menu

### PR DESCRIPTION
**Pull request by Johngtka**
Changes List:

- [x] `removed patients from menu`

obraz

### CODE REVIEW

**Comment by Johngtka** [comment]
s&m commit proposal:
`fix: patients removed from menu (#48)`

**Comment by macrog** [comment]
> s&m commit proposal: `fix: patients removed from menu (#48)`

are you sure this is a "fix"?

File: src/app/app.component.html [hunk 1]
```diff
@@ -26,10 +26,6 @@
             <mat-icon>history</mat-icon>
             <span>{{ 'MENU.VISITS_HISTORY' | translate }}</span>
         </button>
```
Comment (macrog) [inline src/app/app.component.html:28]: plz remove the component as well

**Review by macrog — APPROVED** [review]
👍

**Pull request merged**